modules/security-network-isolating.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-network.adoc
+
+[id="security-network-isolating_{context}"]
+= Isolating applications
+
+{product-title} enables you to segment network traffic on a single cluster to
+make multi-tenant clusters that isolate users, teams, applications, and
```

Comment:
Double check this - I thought that multitenant/multitenancy was one word. Not positive though, we seem to have a mix throughout the doc set

Reply:
It seems to be all over the map. Kubernetes has a "Multitenancy" working group. But documents pointed to from the group's home page have lots of instances of "Multi-tenancy". Our docs have both, but seem to use one word more often. I'll flip a coin..........One word it is. Done